actionpack/test/dispatch/cookies_test.rb

```diff
@@ -1235,6 +1250,23 @@ def test_cookies_are_not_cleared
     assert_equal "bar", @controller.encrypted_cookie
   end
 
+  def test_cookie_expiration_using_expires_in
+    freeze_time do
+      get :set_expiration_using_expires_in
+      assert_cookie_header "user_name=assain; path=/; expires=#{1.hour.from_now.rfc2822}"
```

Comment:
What about timezones here in this test case? Should the `1.hour.from_now` be converted to UTC first?